github-pr

```diff
@@ -12,11 +12,19 @@ def _check_required_fields(required, **args):
             sys.exit(1)
 
 
+def _return_status(commits):
+    commit_statuses = []
+    for commit in commits:
+        for commit_status in commit.get_statuses():
+            commit_statuses.append(commit_status.state)
+    return commit_statuses[-1]
+
 def _print_pr(pr, **args):
     if args['numberonly']:
         print "%d" % pr.number
     else:
-        print "#%d [%s] %10s <- %-30s    %s" % (pr.number, pr.state, pr.base.ref, pr.head.ref, pr.title)
+        commit_status = _return_status(pr.get_commits())
+        print "#%d [%s] %10s <- %-30s    %s - Commit/Build Status:%s" % (pr.number, pr.state, pr.base.ref, pr.head.ref, pr.title, commit_status)
```

Comment:
I mean, if there are 10 prs, we will have `Commit/Build Status: ` on every line. Also, this isn't very parseable, if you try to use awk to split it out, you'll end up with `Status:Success`. Maybe we should print a table header and use something like this https://pypi.python.org/pypi/tabulate to print out a clean, easy to parse table, like what you'd get here:
```
$ ps | head
  PID TTY           TIME CMD
  653 ttys000    0:00.31 -bash
  678 ttys001    0:01.59 -bash
53450 ttys002    0:00.22 -bash
  805 ttys003    0:00.50 -bash
  879 ttys004    0:00.35 -bash
60446 ttys004    0:00.00 head
```